rbac/providers/common/db_queries.py

```diff
@@ -66,21 +66,18 @@ def save_sync_time(provider_id, sync_source, sync_type, timestamp=None):
     conn.close()
 
 
-def peek_at_queue(table_name, provider_id=None):
-    """Returns a single entry from table_name with the oldest timestamp and matching
-    provider_id."""
+def peek_at_queue(table_name):
```

Comment:
This function is used in both LDAP and AAD outbound sync and both workers are trying to get the earliest entry in outbound_queue for their specific provider_id. Without the provider_id, now both outbound syncs are just grabbing the earliest entry from outbound_queue.

Reply:
This is also causing the lint issues.